.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Feature Request
+about: Request a feature addition or change to Capacitor.
+title: 'feat: '
+labels: ''
+assignees: ''
+
+---
+
+# Feature Request
+
+## Describe the Feature Request
+<!-- A clear and concise description of what the feature request is. Please include if your feature request is related to a problem. -->
+
+## Platform Support Requested
+<!-- Leave these checkboxes EMPTY until after the issue is created. Once the issue is created, check one or more boxes for the platform(s) this bug affects. -->
```

Comment:
I copied the bug text for reference, but this one should say feature request instead

Reply:
Oops, copypasta error. My bad.